app/services/author_builder.rb

```diff
@@ -1,34 +1,25 @@
 # frozen_string_literal: true
 
 class AuthorBuilder
-  ALLOWED_ROLES = %w[Author creator].freeze
 
-  # @param [Array<Cocina::Models::Contributor>] contributors
-  # @return [String] the author value for Solr
-  def self.build(contributors)
```

Comment:
changing the methos to build_primary and build_all, instead of trying to make "build" serve both cases